docs/source/delta-uniform.md

```diff
@@ -61,13 +61,25 @@ CREATE TABLE T(c1 INT) USING DELTA TBLPROPERTIES(
   'delta.universalFormat.enabledFormats' = 'iceberg');
 ```
 
-You can enable UniForm Iceberg on an existing table using the following syntax:
+In Delta 3.3 and above, you can enable or upgrade UniForm Iceberg on an existing table using the following syntax:
 
+```sql
+ALTER TABLE table_name SET TBLPROPERTIES(
+  'delta.enableIcebergCompatV2' = 'true',
+  'delta.universalFormat.enabledFormats' = 'iceberg');
+```
+
+You can also use REORG to enable UniForm Iceberg and rewrite underlying data files, as in the following example:
```

Comment:
Should we clarify which Delta versions support using reorg?

Reply:
I would say no need as all versions that support uniform supports REORG...

Reply:
Oops sorry I meant for `ALTER TABLE` I wasn't thinking. SGTM